README.md

```diff
@@ -81,6 +81,18 @@ They are not suitable for most production use-cases because they keep all data i
 You can also install with FerretDB with the `.deb` and `.rpm` packages
 provided for each [release](https://github.com/FerretDB/FerretDB/releases).
 
+## Building and packaging
+
+The best way to build FerretDB is not to do it but use Docker images and `.deb` and `.rpm` packages provided by us.
```

Comment:
That sentence looks a bit toxic to my taste, but maybe I'm wrong. 

@Fashander, from your perspective, is it ok? I understand that the idea is to say that it's better not to build FerretDB and use an existent build if possible, but I'm not sure if the proposed wording sounds friendly.

Reply:
>  the idea is to say that it's better not to build FerretDB

More like strongly discourage

Reply:
It can probably be phrased better. Maybe something like: "We strongly advise users not to build FerretDB. Instead, use the Docker images and `.deb` and `.rpm` packages provided by us."

Reply:
PTAL